src/object.js

```diff
@@ -146,11 +161,40 @@ ReflectionObject.prototype.onRemove = function onRemove(parent) {
 ReflectionObject.prototype.resolve = function resolve() {
     if (this.resolved)
         return this;
+    if (this instanceof Root || this.parent && this.parent.resolved)
+        this._resolveFeatures();
     if (this.root instanceof Root)
-        this.resolved = true; // only if part of a root
+        this.resolved = true;
```

Comment:
Did this change at some point?  I thought we were setting resolved to true for every type of descriptor so that we don't keep re-resolving features?

Reply:
I noticed that `this.root` does that, since it's basically setting it true as long as the object isn't orphaned (vs. `this`)